.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,76 @@
+name: "🐛 Bug Report"
+description: If you've found a reproducible bug
+labels: ["bug"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to file a bug report!
+
+        We're most likely to fix bugs that are easy to reproduce, 
+        so please provide us with detailed steps to reproduce the issue.
```

Comment:
I think they should also check if the issue is not already reported and we can put a link where they should search for it (jira, github issues).

Reply:
done